bookman

```diff
@@ -89,16 +89,16 @@ if [ -n "$SOURCE_DATE_EPOCH" ]; then
 fi
 date=${date:-$(date +'%d %B %Y')}
 
-[[ $1 ]] || set -- $(while read; do echo $REPLY; done)
+[ $1 ] || set -- $(while read REPLY; do echo $REPLY; done)
 
-[[ $outfile ]] && post="$post >$outfile"
+[ $outfile ] && post="$post >$outfile"
 
 {
 	# Compute table of content from postscript output.
 	# Generate output in gtroff intermediate format, so
 	# it can be merged with content.
 	{
-		[ -f "$cover" ] && cat "$cover" || {
```

Comment:
This is not a bashism here. the double quotes protect against spaces in `cover`value. Please do not remove them.